indico/modules/events/management/templates/_settings.html

```diff
@@ -60,6 +60,7 @@
                     {% endif %}
                 {% endcall %}
             </dd>
+            {{ template_hook('extra-event-section-data', event=event) }}
```

Comment:
```suggestion
            {{ template_hook('extra-event-settings-section-data', event=event) }}
```

Reply:
An even nicer alternative would be this (UNTESTED):

```patch
--- a/indico/modules/events/management/templates/_settings.html
+++ b/indico/modules/events/management/templates/_settings.html
@@ -21,6 +21,7 @@
         <div class="icon {{ icon }}"></div>
         <div class="text">
             {{ caller() }}
+            {{ template_hook('event-settings-section-extra', event=event, name=name) }}
         </div>
         <div class="toolbar">
             <button class="i-button borderless text-color icon-only icon-edit hide-if-locked"

```

That way you can add new stuff at the end of any section, so if you need to extend another one you don't need to send another PR

Reply:
Yes this works, thanks!